README.md

```diff
@@ -214,10 +254,43 @@ The `tetra` CLI is also available inside `tetragon` container.
 kubectl exec -it -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact
 ```
 
-Tetragon is able to observe several events, here we provide a few small
-samples that can be used as a starting point:
+## Tetragon Events
+
+Tetragon is able to observe critical hooks in the kernel through its sensors
+and generates enriched events from them. In the next sections we detail the
+available sensors and the events they produce:
+1. [Process execution](#process-execution): generating `process_exec` and
+   `process_exit` events.
+1. [Generic tracing](#tracepoints-and-kprobes): generating `process_kprobes`
+   and `process_tracepoint` events.
+
+Along, we present use cases on how they can be used as a starting point.
 
-### Process Execution ###
+### Process execution
+
+Tetragon observes process creation and termination with default configuration
+and generates `process_exec` and `process_exit` events:
+
+- The `process_exec` events include useful information about the execution of
+  binaries and related process information. This includes the binary image that
+  was executed, command-line arguments, the UID context the process was
+  executed with, the process parent information, the capabilities that a
+  process had while executed, the process start time, the Kubernetes Pod,
+  labels and more.
+- The `process_exit` events, as the `process_exec` event shows how and when a
+  process started, indicate how and when a process is removed. The information
+  in the event includes the binary image that was executed, command-line
+  arguments, the UID context the process was executed with, process parent
+  information, process start time, the status codes and signals on process
+  exit. Understanding why a process exited and with what status code helps
+  understand the specifics of that exit.
+
+These events enable a full lifecycle view into a process that can aid an
+incident investigation, for example, if a suspicious process is still running
```

Comment:
... for example, we can determine if a suspicious process is still running in an environment.